File: CS/T191208/Form1.cs
using System;

namespace T191208
{
	public partial class Form1 : DevExpress.XtraEditors.XtraForm
	{
		public Form1()
		{
			InitializeComponent();
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			this.barManager1.CreateCustomizationForm += barManager1_CreateCustomizationForm;
		}

		private void barManager1_CreateCustomizationForm(object sender, DevExpress.XtraBars.CreateCustomizationFormEventArgs e)
		{
			e.CustomizationForm = new MyForm(new CustomCustomizationControl(), this.LookAndFeel);
		}

	}
}

File: VB/T191208/Form1.vb
Imports System

Namespace T191208
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            AddHandler Me.barManager1.CreateCustomizationForm, AddressOf barManager1_CreateCustomizationForm
        End Sub

        Private Sub barManager1_CreateCustomizationForm(ByVal sender As Object, ByVal e As DevExpress.XtraBars.CreateCustomizationFormEventArgs)
            e.CustomizationForm = New MyForm(New CustomCustomizationControl(), Me.LookAndFeel)
        End Sub

    End Class
End Namespace
